week-2/J-array-get-set/exercise.js

```diff
@@ -5,11 +5,11 @@
 */
 
 function first(arr) {
-  return; // complete this statement
+  return `first value is ${arr[0]} and last array is ${arr[arr.length-1]}`; // complete this statement
```

Comment:
That's not quite what the exercise is asking you to do - it asks you to just return what the first element in the array is, not format that element in a string.

We could then use another function to call `first()` and then format it.